algolia/recommendation/client.go

```diff
@@ -7,11 +7,13 @@ import (
 	"github.com/algolia/algoliasearch-client-go/v3/algolia/transport"
 )
 
+// Deprecated: use personalization.Client instead
 // Client provides methods to interact with the Algolia Recommendation API.
 type Client struct {
 	transport *transport.Transport
 }
 
+// Deprecated: use personalization.NewClient() instead
```

Comment:
Forgoten the Deprecated mention on the next method (`NewClientWithConfig`)

Reply:
Fixed 👌